packages/b/bibletime/pspec_x86_64.xml

```diff
@@ -135,18 +135,1267 @@
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/th/h2-rules.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/th/index.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-basics-approaches.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-basics-expository.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-basics-interpretation.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-basics-types.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-basics-worksheet.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-basics.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-importance-breathed.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-importance-exhortations.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-importance-liberates.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-importance-once.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-importance-supplement.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-importance-wars.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-importance-works.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-importance.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-rules-context.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-rules-hcontest.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-rules-normal.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-rules-parables.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/h2-rules.html</Path>
+            <Path fileType="doc">/usr/share/doc/bibletime/howto/html/uk/index.html</Path>
             <Path fileType="data">/usr/share/icons/hicolor/scalable/apps/info.bibletime.BibleTime.svg</Path>
             <Path fileType="data">/usr/share/metainfo/info.bibletime.BibleTime.metainfo.xml</Path>
         </Files>
     </Package>
     <History>
-        <Update release="1">
-            <Date>2024-05-05</Date>
+        <Update release="2">
+            <Date>2024-05-09</Date>
             <Version>3.0.3</Version>
             <Comment>Packaging update</Comment>
             <Name>Jakob Gezelius</Name>
             <Email>jakob@knugen.nu</Email>
         </Update>
     </History>
-</PISI>
\ No newline at end of file
+</PISI>
```

Comment:
`pspec_x86_64.xml` doesn't usually end with new line. Did you accidentally add them? Can you try rebuild it and push again ?